### ISSUE TITLE
can not reproduce the experimental results in the paper

### ISSUE THREAD

**Issue by YerayL** [vid-koci/KBCtransferlearning#1]
Hi，
    I try to reproduce this result in the paper,
![image](https://user-images.githubusercontent.com/37128324/154916726-eed555ed-139e-42b8-8180-311a664a6c3d.png)
.So I used the same hyperparameter training in the appendix(-data Data/ReVerb20K -dim 300 -lr 3e-3 -batch 1024 -embedding TuckER -dropout 0.4 -encoder GRU).
This is my best result for the test set. Could you tell me what the problem is? Thank you!
02/21/2022 15:59:36 - INFO - __main__ -   MR:   Head 241.96     Tail 1240.93    Overall 741.45
02/21/2022 15:59:36 - INFO - __main__ -   MRR:  Head 0.533      Tail 0.153      Overall 0.343
02/21/2022 15:59:36 - INFO - __main__ -   H@N   1: 0.277        3: 0.370        5: 0.410        10: 0.469       30: 0.558       50: 0.599

**Comment by vid-koci** [comment]
Hi YerayL, does changing the learning rate from 3e-3 to 1e-3 change the results for the better? 3e-3 seems to be a typo in the paper - my old notes say that I got that result with the learning rate 1e-3.

**Comment by YerayL** [comment]
> Hi YerayL, does changing the learning rate from 3e-3 to 1e-3 change the results for the better? 3e-3 seems to be a typo in the paper - my old notes say that I got that result with the learning rate 1e-3.

Thanks for your prompt reply! It is ok after modifying the learning rate.

**Comment by vid-koci** [comment]
Great to hear that it worked. In that case, I'm closing this issue.
